### PR TITLE
Upgrade to dor-services-8

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by jcoyne** [comment]
@andrewjbtw  is testing this on stage right now.

File: config/initializers/config.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Config.setup do |config|
+  config.const_name = 'Settings'
+  config.use_env = true
+  config.env_prefix = 'SETTINGS'
+  config.env_separator = '__'
+  # config.env_converter = :downcase
```
Comment (justinlittman) [inline config/initializers/config.rb]: Cruft?

**Comment by andrewjbtw** [comment]
For testing on stage, I've:

- registered
- accessioned
- versioned
- run some bulk actions 
- run reports
- clicked on and updated datastreams 
- clicked on and viewed workflows

I didn't see any issues with any of these checks.

File: config/initializers/config.rb (hunk 1 above)
Comment (jcoyne) [reply config/initializers/config.rb]: Not really.  This is something we usually do, but we have to convert all the Settings variables to lowercase first. Not something I want to add to this pr.

**Pull request merged**